robottelo/hosts.py

```diff
@@ -2422,24 +2423,44 @@ def run_orphan_cleanup(self, smart_proxy_id=None):
 class SSOHost(Host):
     """Class for RHSSO functions and setup"""
 
-    def __init__(self, sat_obj, **kwargs):
```

Comment:
Not well-thought-through, but would splitting `SSOHost` to `RHSSOHost` and `RHBKHost` make sense here?
```mermaid
graph TD;
    RHBKHost-->SSOHost;
    RHSSOHost-->SSOHost;
    SSOHost-->Host;
```

Reply:
Yes.